templates/chartmuseum/chartmuseum-cm.yaml

```diff
@@ -99,7 +99,11 @@ data:
   STORAGE_ALIBABA_PREFIX: {{ $storage.oss.rootdirectory }}
   {{- end }}
   {{- if $storage.oss.endpoint }}
-  STORAGE_ALIBABA_ENDPOINT: {{ $storage.oss.endpoint }}
+  STORAGE_ALIBABA_ENDPOINT: {{ $storage.oss.endpoint | trimPrefix $storage.oss.bucket | trimPrefix "." }}
```

Comment:
If the `endpoint` is specified, shouldn't we just use it as the `STORAGE_ALIBABA_ENDPOINT` directly?

Reply:
> If the `endpoint` is specified, shouldn't we just use it as the `STORAGE_ALIBABA_ENDPOINT` directly?

yes,but charmuseum endpoint format not include bucket name